### ISSUE TITLE
[FEATURE] template-oss should manage labels

### ISSUE THREAD

**Issue by wraithgar** [npm/template-oss#69]
The settings.yml file inheritance model does not work.  We should let template-oss manage labels, if that's at all possible.

We may end up in a situation like the current one, where inheritance doesn't work. It would mean template-oss itself needs to *build* the settings file from its template any any optional additional file that we allow for in each repo.

**Comment by lukekarrys** [comment]
- [ ] update wiki once this lands https://github.com/npm/cli/wiki/Labelling-Issues-&-PRs

**Comment by nlf** [comment]
| Label | Color | Description |
| --- | --- | --- |
|![Dependencies](https://img.shields.io/badge/-Dependencies-00cc00)|`#00cc00`|Dependency updates|

**Comment by wraithgar** [comment]
the autorelease tags that release-please uses are definitely needed

**Comment by lukekarrys** [comment]
here are all the labels in all of our repos, sorted by how many repos have that label. note this doesn't say anything about how many issues/prs have that label, but just gives us a baseline to not accidentally clobber anything important.

|label name|repo count|
|---|---|
|Dependencies|73|
|autorelease: pending|67|
|autorelease: tagged|66|
|Agenda|56|
|semver:patch|55|
|semver:minor|55|
|semver:major|55|
|Needs Discussion|55|
|Security|55|
|Backlog|54|
|In Progress|54|
|Needs Tests|54|
|Needs Documentation|54|
|Needs Triage|54|
|:gem: Free Internet Points :gem:|54|
|Priority 1|54|
|Priority 2|54|
|Priority 0|54|
|Windows|53|
|Linux|53|
|Needs Review|53|
|Enhancement|48|
|Good First Issue|48|
|Bug|42|
|Wontfix|39|
|Duplicate|39|
|Question|38|
|feature|20|
|invalid|19|
|refactor|18|
|chore|18|
|cd|18|
|docs|18|
|ci|18|
|blocked|18|
|released|18|
|Release 8.x|4|
|Discuss|3|
|Epic|3|
|Meeting|2|
|Documentation|2|
|Registry|2|
|Ratified|2|
|Release 6.x|2|
|Release 7.x|2|
|help wanted|2|
|cli|1|
|spec|1|
|ratify|1|
|ws:libnpmhook|1|
|ws:arborist|1|
|ws:libnpmdiff|1|
|cmd:install|1|
|cmd:publish|1|
|config:include-workspace-root|1|
|config:workspaces|1|
|cmd:pkg|1|
|cmd:list|1|
|config:workspace|1|
|cmd:update|1|
|ws:libnpmexec|1|
|ws:libnpmorg|1|
|ws:libnpmpack|1|
|ws:libnpmpublish|1|
|ws:libnpmaccess|1|
|ws:libnpmsearch|1|
|ws:libnpmteam|1|
|ws:libnpmversion|1|
|cmd:run-script|1|
|cmd:audit|1|
|config:display|1|
|ws:libnpmfund|1|
|cmd:version|1|
|Release 9.x|1|
|config: lockfile|1|
|cmd:exec|1|
|perf|1|
|cmd:diff|1|
|cmd:outdated|1|
|release: manager|1|
|config:global|1|
|config:private-registry|1|
|platform: macos|1|
|Cannot Reproduce|1|
|config:overrides|1|
|cmd:access|1|
|regression:9.x|1|
|cmd:query|1|
|ws:config|1|
|initiative|1|
|provenance|1|
|backport npm9|1|
|linked-install-strategy|1|
|audit-policies-filters|1|
|content|1|
|accessibility|1|
|theme|1|
|Awaiting Information|1|
|Priority Backlog|1|
|pr: needs tests|1|
|pr: needs documentation|1|
|Community|1|
|Website|1|
|Needs Rebase|1|
|Release Next|1|
|platform:windows|1|
|release: next|1|
|platform: linux|1|
